app/views/meals/_form.html.haml

```diff
@@ -0,0 +1,22 @@
+%h3= "#{t :add_meals_to_menu}"
+= form_for [menu, meal] do |f|
+  %p
+    = f.label :name
+    %br
+    = f.text_field :name
+
+  %p
+    = f.radio_button :course, :first_course
+    = f.label :first_course, "First Course"
```

Comment:
No need to specify text for label here, if I remember correctly. Anyway there is a better way to specify custom label text without hardcoding. Explore [here](https://apidock.com/rails/ActionView/Helpers/FormHelper/label)